handlers/mainHandler.py

```diff
@@ -248,9 +249,21 @@ async def _post(self) -> None:
                         if userToken["kicked"]:
                             await tokenList.deleteToken(userToken["token_id"])
 
-                # Release processing lock
-                # if token_processing_lock is not None:
-                #     await token_processing_lock.release()
+                time_elapsed_ms = round((time.perf_counter_ns() - st) / 1000 / 1000, 2)
```

Comment:
isn't this gonna be kinda wrong for every packets beyond the first since its gonna include processing time of prior packets

Reply:
Oh yup let me fix